### PR TITLE
fetched api and used custom hooks

### PR DESCRIPTION
**Pull request by Omarion123**
## What does this PR do?
this helps to fetch Api and use custom hooks for post and get

### CODE REVIEW

File: data/db.json [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
+          "creator": "Eric",
+          "creatorprofile": "https://res.cloudinary.com/dxitrjcef/image/upload/v1696870762/kazdcipwzwu0ycprzlg6.jpg",
+          "PostedDate": "2023-10-18T19:34:16.711Z",
+          "comments": [],
+          "__v": 0,
+          "comment": []
+      }
+  ]
```
Comment (Reconfort) [inline data/db.json:53]: This data must come from API

File: src/components/Dashboard.js [hunk 2]
```diff
@@ -60,66 +64,50 @@ const Dashboard = () => {
...
+            <div>
+              <h2>The blogs is loading.....</h2>
+            </div>
+          </Animated>
+        )}
+        {blogs && (
+          <>
+            {blogs.map((blog) => (
+              <div className="items1" key={blog.id}>
+                {/* <Link to={`/blogs/${blog.id}`}> */}
```
Comment (Reconfort) [inline src/components/Dashboard.js:94]: Remove uneccessary comment

File: src/components/Dashboard.js [hunk 3]
```diff
@@ -60,66 +64,50 @@ const Dashboard = () => {
...
+                </div>
+                <div className="items-description">
+                  {blog && blog.content ? blog.content.substring(0, 100) : ""}
+                  ...
+                </div>
+                <div className="items-action">
+                  <AiFillEdit className="icon" />
+                  <MdDelete className="icon" />
+                </div>
+                {/* </Link> */}
```
Comment (Reconfort) [inline src/components/Dashboard.js:106]: Same case

**Review by Reconfort — CHANGES_REQUESTED** [review]
Review request by the end of today

**Review by mansurissa — APPROVED** [review]

**Review by Reconfort — APPROVED** [review]

**Pull request merged**